src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1931,20 +1970,20 @@ impl DdlController {
 
         // Map the column indices in the dispatchers with the given mapping.
         let downstream_fragments = self.metadata_manager.get_downstream_chain_fragments(id).await?
-                .into_iter()
```

Comment:
The formatter issue still exist? 👀

Reply:
Seems the RustRover has a more "strict" formatter. After I changed back to `rust fmt`, all code looks good to it. So I kept the original format changes introduced by RustRover and not to introduce new ones, as long as it is a large pr.